components/homepage/HomepageNewsFeed.js

```diff
@@ -5,14 +5,27 @@ import Image from 'next/image';
 import styled from 'styled-components';
 import Link from 'next/link';
 import NewsFeed from '../NewsFeed';
+import content from '../../content/newsfeed.md';
 
 // Load images
 import facebook from '../../images/tji-fb-logo-blue.svg';
 import twitter from '../../images/tji-twitter-logo-blue.svg';
 import github from '../../images/tji-github-logo-blue.svg';
 
 class HomepageNewsFeed extends React.Component {
-  render() {
+  render() {git 
```

Comment:
Looks like a stray `git` got committed here. Once you remove this, we should be good to go here!